Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -83,6 +83,15 @@ AzAssetBrowserWindow::AzAssetBrowserWindow(QWidget* parent)
     m_ui->m_assetBrowserTableViewWidget->setVisible(false);
     m_ui->m_toggleDisplayViewBtn->setVisible(false);
     m_ui->m_searchWidget->SetFilterInputInterval(AZStd::chrono::milliseconds(250));
+    m_ui->m_collapseAllButton->setIcon(QIcon(":/stylesheet/img/16x16/layers.png"));
```

Comment:
suggestion: Also might want to experiment with the style settings (by using `applySmallIconStyle` on `AzQtComponents\Components\Widgets\PushButton.h`